htdocs/compta/accounting-files.php

```diff
@@ -318,8 +319,8 @@
 		                	$nofile = array();
 		                	$nofile['id'] = $objd->id;
 		                	$nofile['entity'] = $objd->entity;
-		                	$nofile['date'] = $db->idate($objd->date);
```

Comment:
$objd->date is a string. An when a date value is stored into memory, it must be a timestamp.
So we must use $db->idate() before.

Later if we want timestamp into a string again, we can use dol_print_date()

Reply:
Hello, that is all right, but they are a error in filter that i dont understand why with idate show one day bak the same doc..
![Filter date count files](https://user-images.githubusercontent.com/53640168/95218699-28844280-07ba-11eb-8f71-d99421a90530.jpg)

Reply:
The trouble is due to timezone. Can you try this:
On line 
$date_start = ($date_startDay) ?dol_mktime(0, 0, 0, $date_startMonth, $date_startDay, $date_startYear) : strtotime($date_start);
Replace with 
$date_start = ($date_startDay) ?dol_mktime(0, 0, 0, $date_startMonth, $date_startDay, $date_startYear, true) : strtotime($date_start);

We add the parameter true to dol_mktime so $date_start is a GMT date 
Same with $date_stop = ...

Does it solve your trouble ?

Reply:
Hi, this row in database. While I ll test your recommendation.
![Rowfac674](https://user-images.githubusercontent.com/53640168/95266114-2e991400-07f8-11eb-8e0f-f471099574fe.jpg)

Reply:
And if you add a print $sql in the code, just before it is executed, what do you see as sql request ?

Reply:
Hello the problem this case is timezone. Thankyou